docs/index.rst

```diff
@@ -5552,6 +5552,36 @@ operating points. For ensembles, three kinds of operating points are available:
 number of models in the ensemble that vote for the positive class. The other
 two are already explained in the above mentioned section.
 
+Local Ensemble Predictor
+------------------------
+
+Predictions can take longer when the ensemble is formed by a large number of
+models or when its models have a high number of nodes. In these cases,
+predictions' speed can be increased and memory usage minimized by using the
+``EnsemblePredictor`` object. The basic example to build it is:
+
+.. code-block:: python
+
+    from bigml.ensemblepredictor import EnsemblePredictor
+    ensemble = EnsemblePredictor('ensemble/5143a51a37203f2cf7020351',
+                                 "./model_fns_directory")
+    ensemble.predict({"petal length": 3, "petal width": 1})
+    (u'Iris-versicolor', 0.91519)
+
+This constructor has two compulsory attributes: then ensemble ID (or the
+corresponding API response) and the path to a directory that contains a file
+per each of the ensemble models. Each file stores the ``predict`` function
+needed to obtain the model's predictions. As in the ``Ensemble`` object, you
+can also add an ``api`` argument with the connection to be used to download
+the ensemble's JSON information.
+
+The functions stored in this directory can be generated automatically using
+the command-line tool ``bigmler export``. Please, check the its documentation
+for more details.
```

Comment:
would it be feasible to provide also a function in these same bindings to generate those files, so that everything can be done programmatically if desired? (keeping compatibility with files generated with bigmler, of course... actually, it could even be the case that bigmler used this new functions, since it depends on this bindings).